### ISSUE TITLE
Sertoli cell hypoplasia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11150]
**Preferred term label:**
Sertoli cell hypoplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Same parent as https://hpo.jax.org/browse/term/HP:0034746

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/clinicalSynopsis/608800

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001316
- label: Sertoli cell hypoplasia
- parents: [Abnormal testis morphology (HP:0000035)](https://hpo.jax.org/app/browse/term/HP:0000035)
- pmid: 
- def: Abnormal decrease in the number of Sertoli cells in the testis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11150
- synonyms:
